UltiSnips/python.snippets

```diff
@@ -634,6 +634,11 @@ snippet ipy "Embed IPython shell" b
 import IPython; IPython.embed()
 endsnippet
 
+snippet bpdb "Set bpython/bpdb breakpoint" b
+import bpdb
+bpdb.set_trace()
+endsnippet
```

Comment:
That applies to the others then also, right?
Is `snippets` / snipmate the preferred one?  I would have assumed that `UltiSnips` would be preferred?!